src/resources/postcss/components/skeleton/_tooltip.pcss

```diff
@@ -15,6 +15,8 @@
 
 		.tooltipster-content {
 			padding: 0;
+			overflow-wrap: break-word;
+			word-break: break-all;
```

Comment:
We should almost never use `break-all` for `word-break`, otherwise even just regular sentences that have words that are meant to wrap won't wrap. Use `break-word` instead.

Also if we use `break-word`, do we need `overflow-wrap`?